docs/manual/manual_stow.rst

```diff
@@ -0,0 +1,310 @@
+once it reaches the stow position. Note that incoming commands can cause interruptions to the stow process, as detailed in :ref:`interrupting_stow`.
+
+Module Support
+===============
+
+Speed Modules
+**************
+
+.. table:: Speed Module Support for Stow Position
+
+	+-------------+------------------------------------+
+	| Module      | Stow Position Support              |
+	+-------------+------------------------------------+
+	| Vertiq 8108 | .. centered:: |:white_check_mark:| |
+	+-------------+------------------------------------+
+	| Vertiq 4006 | .. centered:: |:white_check_mark:| |
+	+-------------+------------------------------------+
+	| Vertiq 2306 | .. centered:: |:x:|                |
+	+-------------+------------------------------------+
+
+Servo Modules
+**************
+Servo modules do not support stow as a separate feature. Rather, they are intended to primarily serve as position modules, so controlling them to move to any desired
+position is a natural part of their base functionality. The stow position feature is useful on speed modules primarily as a way to easily allow integrating 
+moving to a single stow position with a flight controller. So the stow specific features described in this portion of the documentation are not relevant for servo modules.
+
+.. table:: Servo Module Support for Stow Position
+
+	+-------------+------------------------------------+
+	| Module      | Stow Position Support              |
+	+-------------+------------------------------------+
+	| Vertiq 8108 | .. centered:: |:x:|                |
+	+-------------+------------------------------------+
+	| Vertiq 4006 | .. centered:: |:x:|                |
+	+-------------+------------------------------------+
+	| Vertiq 2306 | .. centered:: |:x:|                |
+	+-------------+------------------------------------+
+
+.. _stow_process_overview:
+
+Stow Process Overview
+======================
+When a module moves to its stow position, regardles of the reason for the stow, it follows a series of steps to stow properly. First it begins either accelerating or decelerating
+in order to reach its stow position with its user-configrued :ref:`stow acceleration <stow_movement_parameters>`. So if the module was still before, it will begin moving, but if it was 
+spinning rapdily, it willkeep spinning and gradually slow down. Once the module has come to a stop at the stow position, it will then choose whether it should try to 
+actively :ref:`hold that position <stow_holding_position>` or coast itself, meaning that the module will not be driving itself and can spin freely. If the module is :ref:`interrupted <interrupting_stow>` at any time during 
+the stow process, it will stop trying to stow.
+
+The diagram below gives a brief overview of how the stow position feature transitions between different states based on its configuration and how 

```

Comment:
The diagram below summarizes the stow position feature's state transitions. Note that the stow process may be interrupted at any time, and that certain state transitions are configuration dependent.